resources/views/backend/dashboard/index.blade.php

```diff
@@ -6,100 +6,163 @@
+              </div>
             </div>
-        </div>
+            <div class="box-body">
+              <div id="bar-chart" style="height: 300px; margin-top:10px"></div>
+            </div><!-- /.box-body-->
+          </div><!-- /.box -->
+        </div><!-- /.col -->
+      </div><!-- /.row -->
+      <div class="row">
+        <div class="col-md-6">
+          <div class="box">
+            <div class="box-header">
+              <h3 class="box-title">Top 3 User</h3>
+            </div><!-- /.box-header -->
+            <div class="box-body no-padding">
+              <table class="table table-striped">
+
+                <tr>
+                  <th style="width: 10px">#</th>
+                  <th>Name User</th>
+                  <th>Total Order</th>
+                  <th>Date</th>
+                </tr>
+              @foreach ($data['user'] as $key => $value)
+                <tr>
+                  <td>{{ $key }}.</td>
+                  <td>{{ $value->name }}</td>
+                  <td>{{ $value->total }}</td>
+                  <td>{{ $value->date }}</td>
+                </tr>
+              @endforeach  
+              </table>
+            </div><!-- /.box-body -->
+          </div><!-- /.box -->
+        </div><!-- /.col -->
+
+        <div class="col-md-6">
+          <div class="box">
+            <div class="box-header">
+              <h3 class="box-title">Top 3 Product</h3>
+            </div><!-- /.box-header -->
+            <div class="box-body no-padding">
+              <table class="table table-striped">
+                <tr>
+                  <th style="width: 10px">#</th>
+                  <th>Name Product</th>
+                  <th>Total Order</th>
+                  <th>Date</th>
+                </tr>
+              @foreach ($data['product'] as $key => $value)
+                <tr>
+                  <td>{{ $key }}.</td>
+                  <td>{{ $value->name }}</td>
+                  <td>{{ $value->total }}</td>
+                  <td>{{ $value->date }}</td>
+                </tr>
+              @endforeach                
+              </table>
+            </div><!-- /.box-body -->
+          </div><!-- /.box -->
+        </div><!-- /.col -->
+      </div>
+    </section><!-- /.content -->
+  @else
+    <div class="alert alert-info">
+        <strong>Info!</strong> There are no order.
     </div>
+  @endif
 </div>
-@endsection
\ No newline at end of file
+@endsection
+
+@section('script')
+    <script src="{!! asset('assets/backend/plugins/jquery.flot.min.js') !!}"></script>
+    <script src="{!! asset('assets/backend/plugins/jquery.flot.categories.min.js') !!}"></script>
+    <script>
+      $(function () {
+        /*
+         * BAR CHART
+         * ---------
+         */
+        // get array report from controller
+        var arr = <?php echo json_encode($data['order']) ?>;
```

Comment:
[rank B] this variable was declare in view, so you can use {{ ... }} instead echo command